src/modules/event/event.module.ts

```diff
@@ -11,6 +11,12 @@ import { EventAttendees } from '../attendees/entity/attendees.entity';
 import { CohortMember } from './entities/CohortMembers.entity';
 import { Cohort } from './entities/Cohort.entity';
 import { Users } from './entities/Users.entity';
+import { MeetingsService } from '../meetings/meetings.service';
+import { MeetingAdapterFactory } from '../meetings/meetingadapter';
+import { ZoomMeetingAdapter } from '../meetings/adapter/zoom.adapter';
```

Comment:
@poojakarma @vaivk369 This concern needs to solve, we don't want to do core code changes in case of new provider integration